Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -24,6 +24,7 @@
     sys.exit(1)
 
 ami_client = None
+# List for adapting names of AMI values to ES.
```

Comment:
Does adding this comment have anything to do with "Separate request to AMI and adaptation to ES."? 
And maybe it would be better to reword it a little, for example: "Field names in terms of AMI and ES schemes"? At this point it is not very clear what is "AMI values" and what is "adapting"; after reading the code (at least the part where this list is used) one will understand, but comments are usually help to understand things without reading code (or to understand the code itself).

Same question about changing the `change_key_names()` description (and I am not sure if these two changes belong to the same commit, by the way).

Reply:
True, split the commit into several ones, with some rewordings.